common/strings/patch.cc

```diff
@@ -79,7 +79,10 @@ absl::Status MarkedLine::Parse(absl::string_view text) {
 }
 
 std::ostream& operator<<(std::ostream& stream, const MarkedLine& line) {
-  return stream << line.line;
+  term::Color c = line.IsDeleted() ? term::Color::kRed
+                  : line.IsAdded() ? term::Color::kCyan
```

Comment:
(The common color would be green here, but I like cyan as it is more color-blind friendly.)